### ISSUE TITLE
cap production sidekiq:start does not start sidekiq

### ISSUE THREAD

**Issue by prakash-subramani** [seuros/capistrano-sidekiq#231]
I'm using capistrano3. 
ruby: 2.3.5
rails: 5.1.4

while running "production sidekiq:start" seems sidekiq started and pid added to the file. But I didn't see the running process in accctivity monitor or grep. Actually the side sidekiq not running

I have set 'pty' as false in deploy.rb

**Comment by nimish13** [comment]
I am facing the same issue. 
@prakash-subramani  - Did you get any solution for this?

**Comment by prakash-subramani** [comment]
@nimish13 No, I didn't get any solution. I just hold on the process. If you get any solution, please share. 

Thanks!

**Comment by dmitry** [comment]
@prakash-subramani what kind of sidekiq version do you have?

**Comment by dmitry** [comment]
What is the `ps -aux` returns?

**Comment by 7akimz** [comment]
most probably you are missing the 'app' role in your config/deploy/env_name.rb

**Comment by randoum** [comment]
Same issue, with 'app' role correctly configured
